azurepipelines/build/docker/adu-debian-amd64-build.yml

```diff
@@ -1,9 +1,10 @@
 variables:
     version.major: 1
     version.minor: 0
-    version.patch: 2
+    version.patch: 3
```

Comment:
is this patch rev required for the compliant pipeline added support?

Reply:
No I think some files were not updated so I modified them